Source/Controllers/SPAppController.h

```diff
@@ -35,6 +35,8 @@
 @class SPBundleEditorController;
 @class SPWindowController;
 
+@import Firebase;
```

Comment:
Can be only in `m` file?

Reply:
I couldn't get it to work in an .h file. I tried. Needs modules, then get errors. It's a shame as I wanted an app-wide available macro. Maybe you know more about this?